geomesa-raster/src/test/scala/org/locationtech/geomesa/raster/data/RasterStoreQueryIntegratedTest.scala

```diff
@@ -129,7 +129,7 @@ class RasterStoreQueryIntegratedTest extends Specification {
 
       rasterStore must beAnInstanceOf[RasterStore]
       val theResults = rasterStore.getRasters(query).toList
-      theResults.length must beEqualTo(1)
+      theResults.length must beEqualTo(2)
```

Comment:
Agree, this *should* still be 1, each test should create a new mock accumulo table by generating a new table name using getNewIteration.